src/app/features/reporting/query.service.spec.ts

```diff
@@ -82,7 +82,7 @@ describe("QueryService", () => {
     await entityMapper.save(maleChild);
 
     privateSchool = new School("privateSchool");
-    privateSchool.privateSchool = true;
+    privateSchool["privateSchool"] = true;
```

Comment:
maybe we need `EntityConfigService.addConfigAttributes(School)` here so that the privateSchool property will get saved?